code/datums/ai_laws/ai_laws.dm

```diff
@@ -24,18 +24,34 @@
 #define CONFIG_RANDOM 2
 /// Set to a configged weighted list of lawtypes in the config. This lets server owners pick from a pool of sane laws, it is also the same process for ian law rerolls.
 #define CONFIG_WEIGHTED 3
+/// Set to a specific lawset in the game options.
+#define CONFIG_SPECIFIED 4
 
 ///first called when something wants round default laws for the first time in a round, considers config
 ///returns a law datum that GLOB._round_default_lawset will be set to.
 /proc/setup_round_default_laws()
 	var/list/law_ids = CONFIG_GET(keyed_list/random_laws)
+	var/list/spec_law_ids = CONFIG_GET(keyed_list/specified_laws)
 
 	if(HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI))
 		return pick_weighted_lawset()
 
 	switch(CONFIG_GET(number/default_laws))
 		if(CONFIG_ASIMOV)
 			return /datum/ai_laws/default/asimov
+		if(CONFIG_SPECIFIED)
+			var/list/speclaws = list()
+			for(var/lpath in subtypesof(/datum/ai_laws))
```

Comment:
`law_path`

Reply:
Corrected in https://github.com/tgstation/tgstation/pull/68587/commits/21a9bed761bd684beee9787932beeb69b261b2f4